packages/logging_cloudwatch/aws_logging_cloudwatch/lib/src/remote_constraint_provider.dart

```diff
@@ -33,9 +37,83 @@ class DefaultRemoteLoggingConstraintProvider
   final DefaultRemoteConfiguration _config;
   final AWSCredentialsProvider _credentialsProvider;
 
+  LoggingConstraint? _loggingConstraint;
+  DateTime? _lastUpdated;
+
+  Future<void> _saveConstraintLocally(LoggingConstraint constraint) async {
+    final file = File('logging_constraint.json');
```

Comment:
nit: can use const for file name

Reply:
If I switch file to const, I get the error that File doesn't have a const constructor